src/components/templates/Camp/components/MapRoomOverlay.tsx

```diff
@@ -92,10 +92,8 @@ export const MapRoomOverlay: React.FC<MapRoomOverlayProps> = ({
     (e) => {
       e.stopPropagation();
 
+      openRoomUrl(room.url);
       enterCampRoom(room);
```

Comment:
It shouldn't matter since we're opening a new tab, but I think this is better to sit like:
```suggestion
      enterCampRoom(room);
      openRoomUrl(room.url);
```

`enterCampRoom` (a terribly unintuitive name, refactored in 'the great merger' https://github.com/sparkletown/sparkle/pull/840 to something like `trackRoomEntered` or similar) is basically telling firestore they entered, and then we have to 'actually enter' on the client-side (in this case, open the tab)